boards/adafruit/itsybitsy_rp2040/adafruit_itsybitsy_rp2040-pinctrl.dtsi

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2025 Jonas Berg
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ #include <dt-bindings/pinctrl/rpi-pico-rp2040-pinctrl.h>
+
+&pinctrl {
+	uart0_default: uart0_default {
+		group1 {
+			pinmux = <UART0_TX_P0>;
+		};
+
+		group2 {
+			pinmux = <UART0_RX_P1>;
+			input-enable;
+		};
+	};
+
+	i2c1_default: i2c1_default {
+		group1 {
+			pinmux = <I2C1_SDA_P2>;
+			input-enable;
+		};
+
+		group2 {
+			pinmux = <I2C1_SCL_P3>;
+			input-enable;
+		};
```

Comment:
It's a matter of preference, but it's common to group things together if they have the same settings.

Reply:
Done